src/test/run-make-fulldeps/pretty-expanded/input.rs

```diff
@@ -1,6 +1,6 @@
-#[crate_type="lib"]
-
-// #13544
+// FIXME: This should be replaced with some other derive macro
+#![feature(rustc_encodable_decodable)]
+#[crate_type="lib"] // Why is this an outer attribute?
```

Comment:
Not sure what the comment is about, but this should be:

```suggestion
#![crate_type="lib"]
```

Reply:
The comment is because the `#[crate_type="lib"]` line appears in the original version of the test, so I inserted a comment questioning it. It caused my first attempt to fix the test to fail because I saw only the attribute itself and not the relatively insignficant `!` when I added the `#![feature(rustc_encodale_decodable)]` line.

Reply:
Yeah, it's quite strange. I bet the `#[` version of the attribute does nothing, and rustc is just not warning about it.